File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/01_First_Steps_In_Coding_Exercise/03_Deposit_Calculator.js
function depositCalculator(input) {
    let depositAmount = Number(input[0]);  // 200  // "200"
    let depositMonths = Number(input[1]); // 3
    let annualInterestRate222 = Number(input[2]) / 100;  // 0.057

    let finalAmount = depositAmount + depositMonths * ((depositAmount * annualInterestRate222) / 12);

    console.log(finalAmount);
}

// depositCalculator(["200", "3", "5.7"]);

File: Python_Courses/Python_Basics/Python_PB_2023_10/01_First_Steps_In_Coding_Exercise/03_Deposit_Calculator.py
deposit_amount = float(input())
deposit_months = int(input())
annual_interest_rate = float(input()) / 100  # 5.7 => 5.70%

final_amount = deposit_amount + deposit_months * ((deposit_amount * annual_interest_rate) / 12)

print(final_amount)
